src/Amalgam/entity/EntityQueryBuilder.h

```diff
@@ -87,97 +102,90 @@ namespace EntityQueryBuilder
 			[&dist_params](size_t i, bool found, EvaluableNode *en) {
 				if(i < dist_params.featureParams.size())
 				{
-					dist_params.featureParams[i].unknownToUnknownDifference = std::numeric_limits<double>::quiet_NaN();
-					dist_params.featureParams[i].knownToUnknownDifference = std::numeric_limits<double>::quiet_NaN();
```

Comment:
Is this a change in behavior? By default aren't these items 0.0 with your recent changes? Not sure that will cause issues or not.

Reply:
Furthermore, in next section for 'get deviations', they are set to nan. But I could just not understand how they are being used.